src/index.html

```diff
@@ -5,8 +5,328 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class = "header">
+      <a href = "#home" class = "header__logo">
+        <img src = "images/logo.png" alt = "Moyo_logo">
+      </a>
+
+      <nav class = "nav">
+        <ul class = "nav__list">
+          <li class = "nav__item">
+            <a href = "#Apple"  class = "nav__link is-active">Apple</a>
+          </li>
+
+          <li class = "nav__item">
+            <a href = "#Samsung" class = "nav__link">Samsung</a>
+          </li>
+
+          <li class = "nav__item">
+            <a href = "#Smartphones" class = "nav__link">Smartphones</a>
+          </li>
+
+          <li class = "nav__item">
+            <a
+            href = "#Laptops"
+            class = "nav__link"
+            data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class = "nav__item">
+            <a href = "#Gadgets" class = "nav__link">Gadgets</a>
+          </li>
+
+          <li class = "nav__item">
+            <a href = "#Tablets" class = "nav__link">Tablets</a>
+          </li>
+
+          <li class = "nav__item">
+            <a href = "#Photo" class = "nav__link">Photo</a>
+          </li>
+
+          <li class = "nav__item">
+            <a href = "#Video" class = "nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class = "catalog">
+      <div class = "catalog__container">
+        <article class = "card catalog__card" data-qa="card">
+          <img
+            class = "card__img"
+            src = "./images/imac.jpeg"
+            alt = "imac-img"
```

Comment:
You have to use more detailied description for alt atribute.

Reply:
Done